docs/documentation/server_admin/topics/users/con-aia.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// server_admin/topics/users.adoc
+
+[id="con-aia_{context}"]
+= Application initiated actions
+
+Application initiated actions (AIA) allow client application to request user to perform some action on {project_name} side. Usually when OIDC client application
+wants user to login, it redirects him to the login URL as described in the <<con-oidc_{context}, OIDC section>>. After login, user is redirected back to the client application.
+He needs to optionally perform the actions, which were added to his account by administrator as described in the <<proc-setting-required-actions_{context}, previous section>>
+and then being redirected back to the application immediately. However AIA allows client application to request some required actions from the user during login. This can be
+done even if user is already authenticated on the client and has existing SSO session. It is triggered by adding `kc_action` parameter to the OIDC login URL with value containing the requested action.
+For instance `kc_action=UPDATE_PASSWORD` parameter.
+
+NOTE: The `kc_action` parameter is {project_name} proprietary mechanism unsupported by the OIDC specification.
+
+NOTE: Application initiated actions are supported only for OIDC clients.
+
+So if AIA is used, the example flow is like this:
+
+* Client application redirects user to the OIDC login URL with the additional added parameter like for instance `kc_action=UPDATE_PASSWORD`
+
+* There is `browser` flow always triggered as described in the <<_authentication-flows, Authentication flows section>>. In case that user was not authenticated, he needs to authenticate as during normal login.
+In case user was already authenticated before, he might be automatically re-authenticated by SSO cookie without a need to actively re-authenticate and fill his credentials again. In this case, he will be
```

Comment:
```suggestion
In case the user was already authenticated, that user might be automatically re-authenticated by an SSO cookie without needing to actively re-authenticate and supply the credentials again. In this case, that user will be
```